src/opnsense/mvc/app/library/OPNsense/Backup/PHPMailer/Exception.php

```diff
@@ -0,0 +1,39 @@
+<?php
```

Comment:
phpmailer / phpmailer6 packages exists as well, it would be cleaner to use this instead of embedding a fixed version here. :)

Reply:
ok, thanks, will change it